openstack/path_creds.go

```diff
@@ -265,18 +268,25 @@ func (b *backend) userDelete(ctx context.Context, r *logical.Request, d *framewo
 }
 
 func createUser(client *gophercloud.ServiceClient, password string) (*users.User, error) {
+	token := tokens.Get(client, client.Token())
```

Comment:
Maybe this GET can be put in other place to avoid calling it each time temporary user is created?